algolia-crawler.js

```diff
@@ -0,0 +1,134 @@
+// This is a hardcopy of the Algolia crawler configuration for reference.
+// The master lives on Algolias crawler servers and is also executed there.
+// https://crawler.algolia.com/admin/crawlers/1b5189dd-f93c-43b7-8b45-e5ec6b0d319d/configuration/edit
+/* eslint-disable */
+
+new Crawler({
+  appId: 'YOUR-APP-ID',
+  apiKey: 'YOUR-API-KEY',
+  rateLimit: 8,
+  startUrls: ['https://docs.commercetools.com/'],
+  renderJavaScript: false,
+  sitemaps: [],
+  exclusionPatterns: [
+    '**/releases',
+    '**/releases/**',
+    '**.html',
+    '**/**docs/removed-functionality**',
+    '**/**docs/removed-functionality**/**',
+  ],
+  ignoreCanonicalTo: false,
+  discoveryPatterns: ['https://docs.commercetools.com/**'],
+  schedule: 'at 22:00 every weekday',
```

Comment:
Shouldn't it be `every day`?

Reply:
We're not working on weekends,  weekdays are enough.  We get this free from Algolia and I think we should try to use the service carefully in exchange.